src/main/java/com/gregtechceu/gtceu/utils/GTUtil.java

```diff
@@ -612,4 +612,12 @@ public static <T> T getLast(List<T> list) {
     public static <T> ArrayList<T> list(T obj) {
         return new ArrayList<>(List.of(obj));
     }
+
+    public static double min(Double... list) {
+        return Arrays.stream(list).min(Double::compareTo).orElseGet(() -> 0.0);
+    }
+
+    public static double max(Double... list) {
+        return Arrays.stream(list).max(Double::compareTo).orElseGet(() -> 0.0);
+    }
```

Comment:
do this and move these to GTMath
```suggestion
    public static double min(double... values) {
        if (values == null || values.length == 0) throw new IllegalArgumentException();
        if (values.length == 1) return values[0];
        if (values.length == 2) return Math.min(values[0], values[1]);
        double min = Double.MAX_VALUE;
        for (double d : values) {
            if (d < min) {
                min = i;
            }
        }
        return min;
    }

    public static double max(double... values) {
        if (values == null || values.length == 0) throw new IllegalArgumentException();
        if (values.length == 1) return values[0];
        if (values.length == 2) return Math.max(values[0], values[1]);
        double max = Double.MIN_VALUE;
        for (double d : values) {
            if (d > max) {
                max = i;
            }
        }
        return max;
    }
```